t/rose-macro/05-value-values-check.t

```diff
@@ -203,4 +204,46 @@ file_cmp "$TEST_KEY.err" "$TEST_KEY.err" <<__CONTENT__
 __CONTENT__
 teardown
 #-------------------------------------------------------------------------------
+# Check comma escape in "values"
+TEST_KEY="${TEST_KEY_BASE}-comma-esc-1"
+setup
+init_meta <<'__META_CONFIG__'
+[namelist:values_nl1=my_location]
+values = 'Exeter\, England'
+__META_CONFIG__
+run_pass "${TEST_KEY}" rose macro -V --config='../config'
+teardown
+
+TEST_KEY="${TEST_KEY_BASE}-comma-esc-1-comma"
+setup
+init_meta <<'__META_CONFIG__'
+[namelist:values_nl1=my_location]
+values = 'Exeter\, England',
+__META_CONFIG__
+run_pass "${TEST_KEY}" rose macro -V --config='../config'
+teardown
+
+TEST_KEY="${TEST_KEY_BASE}-comma-esc-2"
+setup
+init_meta <<'__META_CONFIG__'
+[namelist:values_nl1=my_location]
+values = 'Exeter\, England', 'Aberdeen\, Scotland'
+__META_CONFIG__
+run_pass "${TEST_KEY}" rose macro -V --config='../config'
+teardown
+
+TEST_KEY="${TEST_KEY_BASE}-comma-esc-bad"
+setup
+init_meta <<'__META_CONFIG__'
+[namelist:values_nl1=my_location]
+values = 'Manchester\, England', 'Glasgow\, Scotland'
```

Comment:
Non-quoted values are already covered by the `my_raw` setting in the original sets of tests.